scripts/coverage_diff.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Install Intel FPGA SDK for OpenCL Pro Edition
+# https://fpgasoftware.intel.com/
+
+# Example usage: python scripts/coverage_diff.py child_build_dir parent_build_dir
+import xml.etree.ElementTree as ET
+import sys
+import argparse
+import glob
+
+
+def want_file(fn):
+    return fn.startswith("./src")
+
+def txt_table(matrix):
+    s = [[str(e) for e in row] for row in matrix]
+    lens = [max(map(len, col))+1 for col in zip(*s)]
+    fmt = '\t'.join('{{:{}}}'.format(x) for x in lens)
+    table = [fmt.format(*row) for row in s]
+    return '\n'.join(table)
+
+def markdown_table(array):
+    """ Input: Python list with rows of table as lists
+               First element as header. 
+        Output: String to put into a .md file 
+        
+    Ex Input: 
+        [["Name", "Age", "Height"],
+         ["Jake", 20, 5'10],
+         ["Mary", 21, 5'7]] 
```

Comment:
Are age and height also strings? `5'10` is not a valid Python expression.

As the example input, why not use (part of) the input data from an actual run.